nix/README.md

```diff
@@ -121,6 +122,42 @@ informations available. They can for example be obtained by running `dune build
 Don't forget to exit and re-enter the editor using this command after switching
 branches, or otherwise changing the dependency tree of Mina.
 
+#### Per-editor instructions
+
+##### Visual Studio Code / vscodium
+
+You have to install the "OCaml Platform" extension, either from
+[official marketplace](https://marketplace.visualstudio.com/items?itemName=ocamllabs.ocaml-platform)
+or [openvsix](https://open-vsx.org/extension/ocamllabs/ocaml-platform).
+
+After installing it, run `code` (or `codium`) from within the `nix develop mina#with-lsp` shell,
+click "Select Sandbox" in the extension menu, and then pick "Global Sandbox". From then on, it should just work.
+
+##### Vim
+
+Install [CoC](https://github.com/neoclide/coc.nvim), and add the following to its configuration (`$HOME/.config/nvim`, or just enter command `:CocConfig`):
+
+```
+{
+  "languageserver": {
+    "ocaml-lsp": {
+      "command": "ocamllsp",
+      "args": [],
+      "filetypes": [
+        "ocaml", "reason"
+      ]
+    }
+  }
+}
+```
+
+Now, whenever you start vim from `nix develop mina#with-lsp`, it should just work.
+
+##### Emacs
+
+You need to install the [tuareg](https://github.com/ocaml/tuareg) and [lsp-mode](https://github.com/emacs-lsp/lsp-mode).
```

Comment:
```suggestion
You need to install [tuareg](https://github.com/ocaml/tuareg) and  a LSP client, like  [lsp-mode](https://github.com/emacs-lsp/lsp-mode) or [eglot](https://github.com/joaotavora/eglot).
```

`eglot` is fairly popular in the Emacs community as well, so much so that it's worth mentioning